python/arcticdb/version_store/_store.py

```diff
@@ -1951,6 +1951,7 @@ def compact_incomplete(
         metadata: Optional[Any] = None,
         prune_previous_version: Optional[bool] = None,
         validate_index: bool = False,
+        upsert_symbol_list: bool = True,
```

Comment:
This shouldn't be a user facing option? For writes, we always write the symbol list key, for appends we only do if there is no version to append to?